### ISSUE TITLE
Can not use o1-mini as there is still an image in the context

### ISSUE THREAD

**Issue by SvenMeyer** [getcursor/cursor#2207]
I used claude-3.5.sonnet before and posted images of screenshots

Now I wanted to switch to `o1-mini` , and although I did not post any images I get the message 

`Trying to submit images without a vision-enabled model selected`

![image](https://github.com/user-attachments/assets/0a74d92e-7cef-4c97-8619-55fd66d2abe0)

**Comment by danperks** [comment]
Have you got chat history above with an image is, as that would be sent in your query?

If not, do you have any images in the "lib" folder? It's possible Cursor might be trying to throw those in, causing this error.

**Comment by SvenMeyer** [comment]
> Have you got chat history above with an image is, as that would be sent in your query?

Yes, as I wrote initially "I used claude-3.5.sonnet before and posted images of screenshots."
Posting an image was a few prompts back, claude did not manage to come up with a good solution for a problem a few prompts later, so I wanted to switch to `o1-mini`.
Which failed.

Ideally some AI API endpoint would not get the images sent if it can't process them anayway .. ?

> If not, do you have any images in the "lib" folder? It's possible Cursor might be trying to throw those in, causing this error.

No

**Comment by ArnarValur** [comment]
Hi. Just curious noob question: Is this maybe also related, this happens to me too when I use the Gemini models (Ive tried them all).

![Image](https://github.com/user-attachments/assets/6e2bd604-9435-425c-a2c0-1368c3da5058)

**Comment by dietcheese** [comment]
Same issue. I'm only including files that are not images. Unsure what I'm missing.

**Comment by ffnpsamwel** [comment]
Logging out of cursor completely and then back in allowed me to continue with an updated model that doesn't support images.

**Comment by jasonlarkin** [comment]
tried logging out and back in, did not solve the issue for me
